roles/ssh_hardening/tasks/hardening.yml

```diff
@@ -50,6 +50,26 @@
   ansible.builtin.include_tasks: crypto_kex.yml
   when: not ssh_kex
 
+- name: "Find host public key files"
```

Comment:
Maybe add a link to some documentation as to "why" we are doing this. I found this: https://www.stigviewer.com/stig/red_hat_enterprise_linux_8/2022-12-06/finding/V-230286